src/transformers/models/gemma/tokenization_gemma.py

```diff
@@ -198,7 +198,7 @@ def _decode(
         else:
             sub_texts = "".join(sub_texts)
 
-        return sub_texts
+        return sub_texts.replace(SPIECE_UNDERLINE, " ")
```

Comment:
lol that's actually a hell lot simpler

Reply:
Exactly! 😅 This also means we don't need to make PRs to all the gemma2 repos 😓 (cc @Vaibhavs10)